### ISSUE TITLE
batch size of 32 do not generate useful segmentation models (64 does)

### ISSUE THREAD

**Issue by dcianferoni** [fxia22/pointnet.pytorch#105]
For some reason when I train using batch size of 32 the accuracy is below 0.5 and visualizing prediction leds to all points predicted with same class. Increasing to 64 leads to good performances. Any explanation?
